doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -78,74 +78,38 @@ class MoveGroupInterfaceTutorial
     // Set the location where text is diplayed to above the robot
     text_pose_.translation().z() = 1.25;
   }
-  /**
-  * Publishes the given message above the robot
-  **/
-  void publishToRViz(std::string msg)
-  {
-    visual_tools_.deleteAllMarkers();
-    // To publish a text message inside Rviz
-    visual_tools_.publishText(text_pose_, msg, rvt::WHITE, rvt::XLARGE);
-    // Batch publishing is used to reduce the number of messages being sent to RViz for large visualizations
-    visual_tools_.trigger();
-    // This prompts the user before continuing.
-    visual_tools_.prompt("Press 'next' in the window to continue");
-  }
-
-  /**
-  * Publishes the given message above the robot, and displays a path
-  **/
-  void publishToRViz(std::string msg, const moveit::planning_interface::MoveGroupInterface::Plan& plan)
-  {
-    // Remove old paths and text
-    visual_tools_.deleteAllMarkers();
-    // To publish a text message inside Rviz
-    visual_tools_.publishText(text_pose_, msg, rvt::WHITE, rvt::XLARGE);
-    // To publish a planed path
-    visual_tools_.publishTrajectoryLine(plan.trajectory_, joint_model_group_);
-    // Batch publishing is used to reduce the number of messages being sent to RViz for large visualizations
-    visual_tools_.trigger();
-    // This prompts the user before continuing.
-    visual_tools_.prompt("Press 'next' in the window to continue");
-  }
 
   /**
   * Publishes the given message above the robot, displays a path, and points
   **/
-  void publishToRViz(std::string msg, const moveit::planning_interface::MoveGroupInterface::Plan& plan, std::vector<geometry_msgs::Pose> points)
+  void publishToRViz(std::string msg, const moveit::planning_interface::MoveGroupInterface::Plan* plan=nullptr, const std::vector<geometry_msgs::Pose>* points=nullptr, bool blocking=true, bool clear_prev=true)
   {
-    // BEGIN_SUB_TUTORIAL visualization
+    // BEGIN_SUB_TUTORIAL visualization2
```

Comment:
if we no longer have a `visualization` we shouldn't have a `visualizatoin2`

Reply:
No we do not